CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
-# Changelog
 
-All notable changes to this project will be documented in this file.
+FEATURES:
```

Comment:
I like that [go-changelog](https://github.com/hashicorp/go-changelog#change-file-formatting) isn't opinionated about the types used but will we validate this for consistency?

Maybe we can update our PR template so it's clear to the PR creator what they need to do?
https://github.com/giantswarm/template/blob/master/docs/pull_request_template.md

Reply:
The types are enforced by the `changelog-pr-body-check` when a PR is created or edited. It will leave a comment if the type is incorrect or if the change is missing. Here is the list https://github.com/hashicorp/go-changelog/blob/master/cmd/changelog-pr-body-check/main.go#L78-L86 I agree that we should provide a PR template including that.

Reply:
Also I forked `go-changelog` in the GS organization. It would be cool to collaborate with Hashicorp on it, but it's also not that much code so we could easily maintain our fork to suit our needs.